src/index.spec.ts

```diff
@@ -1679,9 +1679,15 @@ describe('EppoClient config', () => {
     });
 
     expect(client).toBe(EppoPrecomputedJSClient.instance);
-    td.verify(applicationLogger.error('[Eppo SDK] Invalid precomputed configuration wire'), {
-      times: 1,
-    });
+    td.verify(
+      applicationLogger.error(
+        td.matchers.contains({ err: td.matchers.anything() }),
```

Comment:
Pino API ([link](https://getpino.io/#/docs/api?id=loggererrormergingobject-message-interpolationvalues)) is to have the context object first.